lib/mix/lib/mix/tasks/archive.uninstall.ex

```diff
@@ -8,8 +8,18 @@ defmodule Mix.Tasks.Archive.Uninstall do
 
       mix archive.uninstall archive.ez
 
+  ## Command line options
+
+    * `--sha512` - checks the archive matches the given SHA-512 checksum. Only
+      applies to installations via URL or local path
+
+    * `--force` - forces uninstallation without a shell prompt; primarily
+      intended for automation
   """
+  @switches [
+    force: :boolean
+  ]
```

Comment:
Nitpick, but I would leave a blank line before and after the `@switches` definition.

Reply:
Added